.devcontainer/Dockerfile

```diff
@@ -0,0 +1,72 @@
+# SPDX-FileCopyrightText: 2024 Jev Kuznetsov, written for Adafruit Industries
```

Comment:
the "written for Adafruit Industries" that is found in some files copyright notice is intended for work that was paid for by Adafruit. It's only meant to be used when the author was paid by Adafruit for that contribution.

Reply:
thanks, i'll fix that